File: sheet 2.c
# codeforces-sheet-2
#include<stdio.h>
int main()
{
    int n,i,x,evencount = 0,oddcount = 0,poscount = 0,negcount = 0;
    scanf("%d",&n);
    for(i=1;i<=n;i++){
        scanf("%d",&x);
        if(x%2==0){
            evencount++;
        }
        if(x%2!=0){
            oddcount++;
        }
        if(x>0){
            poscount++;
        }
        if(x<0){
            negcount++;
        }
    }
    printf("Even: %d\n",evencount);
    printf("Odd: %d\n",oddcount);
    printf("Positive: %d\n",poscount);
    printf("Negative: %d\n",negcount);

}

File: sheet 2.R
# codeforces-sheet-2
#include<iostream>
using namespace std;
int main()
{
    int n,m,temp,sum;
    while(1){
        cin>>n>>m;
        if(n>m){
            temp = n;
            n = m;
            m = temp;
        }
        if(n<=0||m<=0){ break;};
        sum = 0;
        for(n;n<=m;n++){
                cout<<n<<" ";
                sum += n;
        }
        cout<<"sum ="<<sum<<endl;
    }
    return 0;
}

